### PR TITLE
Improve test coverage

### PR DESCRIPTION
**Pull request by Exanlv**

### CODE REVIEW

File: tests/Discord/DriverInterfaceTest.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        return [
+            'Plain get' => [
+                'method' => 'GET',
+                'url' => 'http://127.0.0.1:8888',
+            ],
+            'Get with params' => [
+                'method' => 'GET',
+                'url' => 'http://127.0.0.1:8888?something=value',
+                'verify' => [
+                    'args' => ['something' => 'value'],
```
Comment (key2peace) [inline tests/Discord/DriverInterfaceTest.php]: 'args' => $content,

Comment (max-paqt) [inline tests/Discord/DriverInterfaceTest.php:77]:
```suggestion
                    'args' => $content,
```

Comment (Exanlv) [reply tests/Discord/DriverInterfaceTest.php]:
```suggestion
                    'args' => $content,
```

File: tests/Discord/EndpointTest.php [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            // @see https://github.com/discord-php/DiscordPHP-Http/issues/16
+            // 'Params with same prefix, short first' => [
+            //     'uri' => 'something/:thing/:thing_other',
+            //     'replacements' => ['::thing::', '::thing other::'],
+            //     'expected' => 'something/::thing::/::thing other::',
+            // ],
+            // 'Params with same prefix, short first' => [
+            //     'uri' => 'something/:thing_other/:thing',
+            //     'replacements' => ['::thing other::', '::thing::'],
+            //     'expected' => 'something/::thing::/::thing other::',
```
Comment (Exanlv) [inline tests/Discord/EndpointTest.php]:
```suggestion
            //     'expected' => 'something/::thing other::/::thing::',
```

**Review by key2peace — APPROVED** [review]

**Pull request merged**